inc/class-book.php

```diff
@@ -225,6 +225,30 @@ static function getBookInformation( $id = null, $contributors_as_string = true,
 		return $book_information;
 	}
 
+	/**
+	 * Get global, web, pdf and ebook theme options
+	 *
+	 * @return array
+	 */
+	public static function getThemeOptions() : array {
+		$options_classes = [
+			'\Pressbooks\Modules\ThemeOptions\GlobalOptions',
+			'\Pressbooks\Modules\ThemeOptions\WebOptions',
+			'\Pressbooks\Modules\ThemeOptions\PDFOptions',
+			'\Pressbooks\Modules\ThemeOptions\EbookOptions',
+		];
+		$theme_options = [];
+		foreach ( $options_classes as $option_class ) {
+			$slug = call_user_func( $option_class . '::getSlug' );
+			$options = get_option( 'pressbooks_theme_options_' . $slug );
+			if ( $options ) {
+				$theme_options[ $slug ] =
+					( new $option_class( $options ) )->options;
+			}
+		}
+		return $theme_options;
```

Comment:
This could also be changed to a more function style, not required though:

```php
return array_reduce( $options_classes, static function( $theme_options, $option_class ) {
	$slug = call_user_func( $option_class . '::getSlug' );
	$options = get_option( 'pressbooks_theme_options_' . $slug );

	return $options ? array_merge( $theme_options, [ $slug => ( new $option_class( $options ) )->options ] ) : $theme_options;
}, [] );
```